blocks/cards/cards.css

```diff
@@ -78,9 +78,13 @@ main .block.cards {
   margin: 0 auto;
   text-align: left;
   color: var(--light-black);
-  padding: 20px;
 }
 
+.cards.grantscard .cards-card-body{
```

Comment:
How about we give this variant a more generic name like "stacked-cards" or something so that it is more readable when reused in a non-grants context?

Reply:
Incorporated. Thanks